libraries/eventformatter/impl/src/main/kotlin/io/element/android/libraries/eventformatter/impl/DefaultRoomLastMessageFormatter.kt

```diff
@@ -95,7 +95,9 @@ class DefaultRoomLastMessageFormatter @Inject constructor(
             is StateContent -> {
                 stateContentFormatter.format(content, senderDisplayName, isOutgoing, RenderingMode.RoomList)
             }
-            is PollContent, // TODO Polls: handle last message
+            is PollContent -> {
+                prefixIfNeeded(content.question, senderDisplayName, isDmRoom)
```

Comment:
Don't we want to prefix the question with something like "Poll: ", or even better display a picto?

Reply:
Nice idea, forwarded the question to desgin: https://github.com/vector-im/element-meta/issues/2048#issuecomment-1727894105
Not sure though how easy it is to include an icon/image here with the current code that handle the last messages.